cognite/client/data_classes/simulators/logs.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Any, Literal
+
+from typing_extensions import Self
+
+from cognite.client.data_classes._base import (
+    CogniteObject,
+    CogniteResource,
+    CogniteResourceList,
+    IdTransformerMixin,
+)
+
+if TYPE_CHECKING:
+    from cognite.client import CogniteClient
+
+SEVERITY = Literal["Debug", "Information", "Warning", "Error"]
+
+
+@dataclass
+class SimulatorLogData(CogniteObject):
+    timestamp: int
+    message: str
+    severity: str
+
+
+class SimulatorLog(CogniteResource):
+    def __init__(
+        self,
+        id: int,
+        severity: str,
+        data: list[SimulatorLogData],
+        created_time: int,
+        last_updated_time: int,
+        data_set_id: int,
+    ) -> None:
+        self.id = id
+        self.severity = severity
+        self.created_time = created_time
+        self.last_updated_time = last_updated_time
+        self.data_set_id = data_set_id
+        self.data = data
+
+    @classmethod
+    def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None = None) -> Self:
+        return cls(
+            id=resource["id"],
+            severity=resource.get("severity", "Debug"),
```

Comment:
Is this field always returned by the API or not? If not, does a missing severity field always mean it's a `debug` log?

Consider if any of these are more appropraite:

Making `severity` optional, 
```suggestion
            severity=resource.get("severity"),
```

having a "Unknown" severity (in that case the Severity literal should be extended)
```suggestion
            severity=resource.get("severity", "Unknown"),
```

or if the API always returns the field, then just
```suggestion
            severity=resource["severity"],
```